composer.json

```diff
@@ -68,7 +68,7 @@
         "friendsofsymfony/oauth-server-bundle": "^1.5.2",
         "stof/doctrine-extensions-bundle": "^1.2",
         "scheb/two-factor-bundle": "^2.14.0",
-        "grandt/phpepub": "^4.0.7",
+        "grandt/phpepub": "dev-master",
```

Comment:
Are you sure you're using @Daniel-KM's PHPePub?

Reply:
Yup, because of https://github.com/wallabag/wallabag/pull/3758/files#diff-b5d0ee8c97c7abd7e3fa29b9a27d1780R147

Reply:
Oh I see, the package name don't change.